frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/import/import_panel.dart

```diff
@@ -162,6 +132,55 @@ class ImportPanel extends StatelessWidget {
 
     importCallback(importType, '', null);
   }
+
+  Widget _buildKeyboardListeningFlowyContainer(
+    BuildContext context,
+    double height,
+    double width,
+  ) {
+    return KeyboardListener(
+      autofocus: true,
+      focusNode: FocusNode(),
```

Comment:
FocusNode needs to be disposed, turn this widget into a StatefulWidget, and add a dispose method:
```dart
  final focusNode = FocusNode();

  @override
  void dispose() {
    focusNode.dispose();
    super.dispose();
  }

  @override
  Widget build(BuildContext context) {
    ...
```